tests/core/gui/test_create_organization.py

```diff
@@ -0,0 +1,248 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+
+from PyQt5 import QtCore
+
+from tests.common import customize_fixtures
+
+from parsec.api.protocol import OrganizationID
+from parsec.core.types import BackendOrganizationBootstrapAddr
+
+
+@pytest.fixture
+def catch_create_org_widget(widget_catcher_factory):
+    return widget_catcher_factory("parsec.core.gui.create_org_widget.CreateOrgWidget")
+
+
+@pytest.fixture
+async def organization_bootstrap_addr(running_backend):
+    org_id = OrganizationID("AnomalousMaterials")
+    org_token = "123456"
+    await running_backend.backend.organization.create(org_id, org_token)
+    return BackendOrganizationBootstrapAddr.build(running_backend.addr, org_id, org_token)
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+@pytest.mark.parametrize("backend_status", [True, False])
+@customize_fixtures(backend_spontaneous_organization_boostrap=True)
+async def test_create_organization(
+    gui,
+    aqtbot,
+    running_backend,
+    qt_thread_gateway,
+    catch_create_org_widget,
+    autoclose_dialog,
+    backend_status,
+):
+
+    if backend_status:
+        gui.config = gui.config.evolve(default_backend_addr=str(running_backend.addr))
+    else:
+        gui.config = gui.config.evolve(default_backend_addr="parsec://nonexisting.com:6666")
+
+    await aqtbot.key_clicks(gui, "n", QtCore.Qt.ControlModifier)
```

Comment:
What is this for ? (I guess it's for triggering a shortcut, but it's not that clear so a comment would be welcome ^^)